backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -204,10 +213,33 @@ export class ContributionResolver {
 
     // all possible cases not to be true are thrown in this function
     validateContribution(creations, amount, creationDateObj)
+
+    const contributionMessage = ContributionMessage.create()
+    contributionMessage.contributionId = contributionId
+    contributionMessage.createdAt = contributionToUpdate.updatedAt
+      ? contributionToUpdate.updatedAt
+      : contributionToUpdate.createdAt
+    let changeMessage = ''
+    if (contributionToUpdate.contributionDate.getTime() !== new Date(creationDate).getTime()) {
+      changeMessage += contributionToUpdate.contributionDate + '\n'
+    }
+    if (contributionToUpdate.memo !== memo) {
+      changeMessage += '---\n' + contributionToUpdate.memo + '\n'
+    }
+    if (contributionToUpdate.amount.toFixed(6) !== amount.toFixed(6)) {
+      changeMessage += '---\n' + contributionToUpdate.amount
+    }
```

Comment:
```suggestion
    const changeMessage = `${creationDate}
---
${memo}
---
${amount.toFixed(6)}`
```
I suggest to always show the whole content, not only the changed. Why `toFixed(6)`.